Automattic/woocommerce-payments
# feat(Settings): add sortable list reusable component

Fixes  https://github.com/Automattic/woocommerce-payments/issues/1504

#### Changes proposed in this Pull Request









- Added Drag & Drop sorting library ( paJDYF-1la-p2 )
- Introduced D&D on WCPay Settings redesign

#### Testing instructions





- Ensure you updated your dependencies with `npm install`
- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Visit http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- You should now be able to Drag & Drop the (mocked) list of payment methods
- Please note that sorting order does not save on refresh

![2021-04-21 16 29 44](https://user-images.githubusercontent.com/273592/115623465-d79feb80-a2be-11eb-956e-0b3112674cef.gif)

I created https://github.com/Automattic/woocommerce-payments/issues/1652 to address the situation where when dragging the first item to the bottom, it appears to be below the second item.


-------------------
